apps/web/src/server/views/applications/components/file-uploader.component.njk

```diff
@@ -0,0 +1,28 @@
+{% macro fileUploader() %}
+	<div class="pins-file-upload">
+		<div class="govuk-body colour--secondary">
+			<p>
+				Choose a single or multiple files to upload.
+			</p>
+			<p>
+				Your file must be PDF, DOC, DOCX, PPT, PPTX, XLS, XLSX, JPG, JPEG, MPEG,
+				MP3, MP4, MOV, PNG, TIF, TIFF or ZIP.
+			</p>
+			<p>
+				The total of your uploaded files must be smaller than 1 GB.
+			</p>
+		</div>
+		<div class="display--flex">
+			<input class="display--none" type="file" id="upload-input" name="files" value="Choose file"
```

Comment:
worth adding the allowed filetypes now with `accept` attibute - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept

Reply:
will add this in the file types subtask